draft-enghardt-panrg-path-properties.md

```diff
@@ -91,63 +92,62 @@ The notion of reliability depends on the property, it might be the confidence le
+Some path properties relate to the entire path, part of which often lies outside of an endpoint's domain. Thus, such properties are listed as Backbone Properties.
+
+
+Presence of a certain network function on the path:
+: Indicates that a certain path element performs a certain network function on a flow, e.g., whether the path element acts as a proxy, as a firewall, or performs Network Address Translation (NAT). This path element may be either in the same domain as the endpoint or in a different domain, i.e., the backbone.
+
+Administrative Entity:
+: The administrative entity, e.g., the AS, to which a path element or set of path elements belongs.
 
 Disjointness:
-: How disjoint a path is from another path.
+: For a set of two paths, the number of shared path elements can be a measure of intersection (e.g., Jaccard coefficient, which is the number of shared elements divided by the total number of elements). Conversely, the number of non-shared path elements can be a measure of disjointness (e.g., 1 - Jaccard coefficient). A multipath protocol might use disjointness of paths as a metric to reduce the number of single points of failure.
 
 Path MTU:
-: The maximum size, in octets, of an IP packet that can be transmitted without fragmentation on a path segment or path.
+: The maximum size, in octets, of an IP packet that can be transmitted without fragmentation on a path segment.
 
 Transport Protocols available:
-: Whether a specific transport protocol can be used to establish a connection over this path. An endpoint may know this because it has cached whether it could successfully establish, e.g., a QUIC connection, or an MPTCP subflow.
+: Whether a specific transport protocol can be used to establish a connection over a path or path segment. An endpoint may cache its knowledge about recent successfully established connections using specific protocols, e.g., a QUIC connection, or an MPTCP subflow, over a specific path.
 
 Protocol Features available:
-: Whether a specific feature within a protocol is known to work over this path, e.g., ECN, or TCP Fast Open.
+: Whether a specific protocol feature is available over this path, e.g., Explicit Congestion Notification (ECN), or TCP Fast Open.
 
 
 # Dynamic Properties
 
-Dynamic Path Properties are expected to change on the timescale of milliseconds.
-They usually relate to the state of the path, such as the current end-to-end latency.
-Some of these properties may depend only on the first hop or on the access network, some may depend on the entire path.
-Properties related to a single layer 2 domain are abstracted from the used physical and link layer technology, similar to {{RFC8175}}.
+Dynamic path properties relate to a path segment with respect to the transmission of an individual packet or of a flow over this path segment.
+Properties related to a path element which constitutes single layer 2 domain are abstracted from the used physical and link layer technology, similar to {{RFC8175}}.
```

Comment:
Properties related to a path element which constitute a single layer 2 domain are abstracted from the used physical and link layer technology, similar to {{RFC8175}}.

Reply:
Fixed.